### DISCUSSION TITLE
heroStyle in Front Matter doesn't work

### DISCUSSION THREAD

**Discussion by Lin-Nanai** [nunocoracao/blowfish#473] [Q&A]
first，I set `article.showHero = true` &  `article.heroStyle = "basic"`  in /config/_default/params.toml

and then in one article, I add `showHero: true; heroStyle: "background"` to Front Matter, also put the background.jpg in the folder

but after `hugo server` , I can't see background, the article page still "basic"

it's something I did wrong?

**Comment by nunocoracao** [comment]
Hello @Lin-Nanai have you tried using a name like "cover.*" or "feature.*" for the image file
